core/src/main/java/com/epam/pipeline/entity/datastorage/AbstractDataStorage.java

```diff
@@ -86,6 +86,11 @@ public abstract class AbstractDataStorage extends AbstractSecuredEntity {
     private Set<String> linkingMasks;
     private Long sourceStorageId;
 
+    /**
+     * Indicates that permissions can be granted for particular paths in this storage.
+     */
+    private boolean pathPermissionsEnabled;
```

Comment:
Let's also add this new field to `cloud-pipeline-common` package

Reply:
added